server/entities/User.ts

```diff
@@ -25,10 +25,27 @@ export class User {
     @Column('varchar', { nullable: true })
     familyName: string | null;
 
+    @Column('varchar', { nullable: true })
+    biography: string | null;
+
     @Column('int', { length: 2 })
     role: UserRole = UserRole.Student;
 
     get roleName() {
         return UserRole[this.role];
     }
+
+    get url() {
+        return '/profile/' + this.id
+    }
+
+    set bio(bio: string) {
+        if(bio && bio.length >= 250) {
+            this.bio = bio;
+        }
+    }
+
+    userCanModify(user: User): boolean {
+        return this.id == user.id || user.role == UserRole.Admin;
```

Comment:
Teachers should probably be able to modify also (in case somebody puts bad stuff in there). So for that, you would probably want to compare with >=